Makefile

```diff
@@ -45,12 +45,12 @@ MAKEFLAGS += --no-print-directory
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 HOSTGO := env -u GOOS -u GOARCH -u GOARM -- go
-
-LDFLAGS := $(LDFLAGS) -X main.commit=$(commit) -X main.branch=$(branch) -X main.goos=$(GOOS) -X main.goarch=$(GOARCH)
```

Comment:
Is there any harm in leaving the goos and goarch values? It does make output a bit more verbose to ensure those settings are getting passed in.

Reply:
> Or should I just add variables to LDFLAGS and don't add variables to the package?

Please I think that is the clearest way to go.
Thanks again!

Reply:
Why weren't main.goos and main.goarch causing lint failures before this PR? It doesn't look like this PR removed the last usage of them.

Reply:
It's probably because the linter runs `lint-branch`, i.e. only checks the modified parts. As probably nobody touched this section for very long, there was no issue reported.